www/src/components/guidelines/color/swatch.js

```diff
@@ -0,0 +1,121 @@
+import React from "react"
+import CopyToClipboard from "react-copy-to-clipboard"
+
+import { Box } from "../system"
+
+export default class Swatch extends React.Component {
+  state = {
+    value: this.props.color.hex,
+    displayCopied: false,
+  }
+
+  toggleCopied = () => {
+    this.setState({
+      displayCopied: true,
+    })
+
+    setTimeout(() => {
+      this.setState({
+        displayCopied: false,
+      })
+    }, 2500)
+  }
+
+  handleClick = event => {
+    event.preventDefault()
+    event.stopPropagation()
+  }
+
+  render() {
+    const { accessibilityLabel, color, swatchStyle, textColor } = this.props
+
+    return (
+      <Box
+        borderRadius={1}
+        bg={color.hex}
+        css={{
+          ...swatchStyle,
+          ":hover > .btn-copy": {
+            display: `block`,
+          },
+        }}
+      >
+        {accessibilityLabel !== `×` && (
+          <Box
+            color={textColor}
+            fontSize={0}
+            position="absolute"
+            fontWeight={0}
+            css={{
+              lineHeight: 1,
+              bottom: 2,
+              top: `auto`,
+              left: 3,
+            }}
+          >
+            {accessibilityLabel}
+          </Box>
+        )}
+
+        <CopyToClipboard text={this.state.value} onCopy={this.toggleCopied}>
+          <button
+            className="btn-copy"
+            tabIndex="0"
```

Comment:
You shouldn't need tabIndex here, as buttons are natively focusable.

Reply:
Oooohh, thank you Marcy! I had that in there because I _tried_ to make the copy buttons tab-able, working around the `display: none` which I understand prevents that. But the `tabIndex` didn't help with that, either. Switching to `opacity` does, though, and allows to remove `tabIndex`.